### ISSUE TITLE
Cpu autodetection

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1667]
**Reported by ilvalle on 2 Feb 2016 20:19 UTC**

On a ubuntu with kernel 4.2.0-27-generic, I got the following
```
$$ aircrack-ng -u
Vendor          = Intel
Model           = Intel(R) Core(TM) i5-2450M CPU @ 2.50GHz
Features        = MMX,SSE,SSE2,SSE3,SSSE3,SSE4.1,SSE4.2,AES-NI,AVX
Hyper-Threading = No
CPU cores       = 4
SIMD size       = 4 (128 bit)
```
However according to http://ark.intel.com/products/53452/Intel-Core-i5-2450M-Processor-3M-Cache-up-to-3_10-GHz Hyper-Threading is supported

**Comment by aircrack-ng** [comment]
**Comment by misterx on 2 Feb 2016 20:19 UTC**

Have you checked if it isn't disabled in the BIOS?

**Comment by aircrack-ng** [comment]
**Comment by misterx on 2 Feb 2016 20:19 UTC**

Should be fixed by aircrack-ng/aircrack-ng@4472d7dec98133e6f0df336457f2caf1c73d9842.
